desktop/extensions-sdk/dev/test-debug.md

```diff
@@ -68,7 +70,7 @@ If your extension is composed of one or more services running as containers in t
 
 ## Update the extension
```

Comment:
this also belongs to the end of the tutorial. 
Maybe it's simpler to keep the test-debug page as the last step of the tutorial, and part of the flow, instead of moving it in the SDK tools section? (although I like the idea initially, it seems that there are several places where it's not ideal).

Reply:
Same thing below with the 'what's next section, it belongs to the tutorial

Reply:
I have added more mention to the update command in the tutorial and removed it from here.

Reply:
Should be better in [ae6e58b](https://github.com/docker/docs/pull/16290/commits/ae6e58bc200fdb9fd9bfa38be30d43b0917fc0ec)